.github/ISSUE_TEMPLATE/implementing-new-platform.md

```diff
@@ -0,0 +1,87 @@
+# Implementing a new supported platform
+
+ ## At Development
+ Create PR's addressing the following:
+ 
+ - [ ] [stream-metadata-go](https://github.com/coreos/stream-metadata-go)
+     - [Example PR](https://github.com/coreos/stream-metadata-go/pull/45/)
+     - [ ] Add platform to the Media struct in release/release.go
+     - [ ] Add supporting code for new platform to _toStreamArch_ func found in the release/translate.go
```

Comment:
```suggestion
     - [ ] Add supporting code for new platform to `toStreamArch` func in `release/translate.go`
```

Reply:
The underscores are still there.

Reply:
Also it looks like the last set of backticks didn't happen.

Reply:
welp dang it, sorry thats what I get for multitasking.. :( should be fixed now.